src/biokbase/narrative/jobs/appmanager.py

```diff
@@ -780,7 +756,12 @@ def _generate_input(self, generator):
         """
         symbols = 8
         if 'symbols' in generator:
-            symbols = int(generator['symbols'])
+            try:
+                symbols = int(generator['symbols'])
```

Comment:
I figure there are two types of things to check here. I let the int() function test if that's actually an integer string (and raise a ValueError if the int() function fails). Then, once it succeeds to make an int, line 763 tests that it meets the minimum value.